src/helpers/geoChart/lngLatBounds.ts

```diff
@@ -0,0 +1,49 @@
+// (C) 2020 GoodData Corporation
+import { IGeoLngLatBounds, IGeoLngLatLike } from "../../interfaces/GeoChart";
+
+/*
+ * @method getLngLatBounds: IGeoLngLatBounds
+ * Represents a rectangular geographical area on a map.
+ *
+ * @example
+ *
+ * ```js
+ * const corner1 = [40.712, -74.227],
+ * const corner2 = [40.774, -74.125],
+ * const bounds = getLngLatBounds([corner1, corner2]);
+ *
+ * bounds && map.fitBounds([bounds.northEast, bounds.southWest], { padding: 60 });
+ * ```
+ */
+export function getLngLatBounds(lnglats: IGeoLngLatLike[]): IGeoLngLatBounds {
+    if (!lnglats || !lnglats.length) {
+        return;
+    }
+
+    let bounds: IGeoLngLatBounds = extendLngLatBounds(undefined, lnglats[0]);
+    for (let i = 1, len = lnglats.length; i < len; i++) {
+        bounds = extendLngLatBounds(bounds, lnglats[i]);
+    }
+
+    return bounds;
+}
```

Comment:
We can use `reduce` to be more functional :)
```
export function getLngLatBounds(lnglats: IGeoLngLatLike[]): IGeoLngLatBounds {
    if (!lnglats || !lnglats.length) {
        return;
    }

    return lnglats.reduce(extendLngLatBounds, undefined);
}
```

Reply:
It's perfect.
Thank you

Reply:
updated